docs/docs/frigate/installation.md

```diff
@@ -243,17 +243,29 @@ you can run Frigate NVR on DSM 7.x. tested on DSM 7.1.1-42962 Update 4
 
 
 General:
+you need to tick the "Execute container using high privilage" checkbox in order to give the frigate container the elevated privilages it may need.
+if you want the container to automatically restart whenever it improperly shuts down then you can tick the "Enable auto-restart" checkbox.
 
-![image](https://user-images.githubusercontent.com/4516296/232573326-0114c001-f5f8-4c62-863b-95c0240920bd.png)
+![image](https://user-images.githubusercontent.com/4516296/232586790-0b659a82-561d-4bc5-899b-0f5b39c6b11d.png)
 
 Advanced Settings:
+you need to add the "FRIGATE_RTSP_PASSWORD" environment variable and set it to your preferred password under advanced settings the rest of the environment variables you can leave as default
 
-![image](https://user-images.githubusercontent.com/4516296/232573502-9c0cdeee-b4e2-4c20-b5c3-d2ba3ebf0065.png)
+![image](https://user-images.githubusercontent.com/4516296/232587163-0eb662d4-5e28-4914-852f-9db1ec4b9c3d.png)
 
-Port Settings (whatever ports you prefer):
 
-![image](https://user-images.githubusercontent.com/4516296/232573652-97c73788-4283-422a-bc60-7fdd2755d68e.png)
+Port Settings:
+the network should be set to a "bridge" network. You need to map the default frigate container ports to your local synology NAS ports that you want to access Frigate using.
+> sometime the defaults container ports that frigate uses my not be available
+> on your NAS due to services you may be running, 
+> in that instance you can set the ports to auto or whatever ports you prefer
+
+![image](https://user-images.githubusercontent.com/4516296/232582642-773c0e37-7ef5-4373-8ce3-41401b1626e6.png)
 
 Volume Settings:
+under volumes you need to configure 2 paths
+- The location of your config file in yaml format, this needs to be file and you need to go to the location of where your config.yml is located, this will be different depending on your NAS folder structure e.g. /docker/frigate/config/config.yml will mount to /config/config.yml within the container
+- The location on your NAS where the recordings will be saved this needs to be a folder e.g. /docker/volumes/frigate-0-media (or where ever you would like the recordings to get saved on your NAS
+
+![image](https://user-images.githubusercontent.com/4516296/232585872-44431d15-55e0-4004-b78b-1e512702b911.png)
```

Comment:
```suggestion
Volume Settings:

You need to configure 2 paths:
- The location of your config file in yaml format, this needs to be file and you need to go to the location of where your config.yml is located, this will be different depending on your NAS folder structure e.g. `/docker/frigate/config/config.yml` will mount to `/config/config.yml` within the container.
- The location on your NAS where the recordings will be saved this needs to be a folder e.g. `/docker/volumes/frigate-0-media`

![image](https://user-images.githubusercontent.com/4516296/232585872-44431d15-55e0-4004-b78b-1e512702b911.png)
```